lib/linguist/heuristics.yml

```diff
@@ -368,7 +368,7 @@ disambiguations:
   - language: Assembly
     pattern:
     - '(?i)mov\s+[^\s]+,'
-    - '^\s+(i?)db\s+[\d\w]'
+    - '^\s+(i?)db\s+[a-z\d]'
```

Comment:
So just to confirm, this should definitely not match any upper case letters?

Reply:
The `i` modifier handles it